src/test/java/org/jenkinsci/plugins/github/config/GitHubServerConfigIntegrationTest.java

```diff
@@ -5,18 +5,16 @@
 import com.cloudbees.plugins.credentials.CredentialsScope;
 import com.cloudbees.plugins.credentials.CredentialsStore;
 import com.cloudbees.plugins.credentials.domains.Domain;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import net.sf.json.JSONObject;
 import org.htmlunit.HttpMethod;
 import org.htmlunit.Page;
 import org.htmlunit.WebRequest;
 import hudson.security.GlobalMatrixAuthorizationStrategy;
 import hudson.util.Secret;
 import jenkins.model.Jenkins;
-import net.sf.json.JSONObject;
-import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.server.ServerConnector;
-import org.eclipse.jetty.ee9.servlet.DefaultServlet;
```

Comment:
No more references to a specific EE version, making this test robust against future EE upgrades (including EE 10).